### PR TITLE
new readme, new method, bonus challenge

### PR DESCRIPTION
**Pull request by SophieDeBenedetto**
@jmburges

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,74 @@
...
+
+### Using the `application.yml` file
+
+In this lab, you'll be storing your API keys in an `application.yml` file. Once we learn about developing applications for the web, you'll get into the habit of storing data that should be available across your applications files in environment variables in YAML files. Another benefit of storing data like API keys and secrets in a separate a file is to protect them from scary internet users. If we keep our keys in a separate file and add that file to our `.gitignore` file, that file *won't be pushed up to Github* and our secret keys and passwords will be safe from hackers!
+
+**Top Tip:** A **`.gitignore`* file is a manifest of files that you don't want to push up to Github. To tell prevent certain files from being pushed up to Github, simply add them to a `.gitignore` file. 
 
 ## Instructions
```
Comment (jmburges) [inline README.md:22]: do they understand bundle and gems and all that?

**Comment by jmburges** [comment]
I think we talked about this, but your tests still only check that the appropriate class gets returned. It doesn't also check that the data returned is the correct data.

**Comment by SophieDeBenedetto** [comment]
https://github.com/learn-co-curriculum/twitter-api-todo/blob/master/spec/twitter_spec.rb

i think this spec is testing for class types and exact values?

**Comment by jmburges** [comment]
yeah you're right, I was looking at the wrong thing

**Pull request closed without merging**